code/game/machinery/ARES/ARES_interface_admin.dm

```diff
@@ -0,0 +1,494 @@
+			admin_interface.current_menu = "access_log"
+		if("page_admin_list")
+			admin_interface.last_menu = admin_interface.current_menu
+			admin_interface.current_menu = "admin_access_log"
+		if("page_security")
+			admin_interface.last_menu = admin_interface.current_menu
+			admin_interface.current_menu = "security"
+		if("page_requisitions")
+			admin_interface.last_menu = admin_interface.current_menu
+			admin_interface.current_menu = "requisitions"
+		if("page_flight")
+			admin_interface.last_menu = admin_interface.current_menu
+			admin_interface.current_menu = "flight_log"
+		if("page_emergency")
+			admin_interface.last_menu = admin_interface.current_menu
+			admin_interface.current_menu = "emergency"
+		if("page_deleted")
+			admin_interface.last_menu = admin_interface.current_menu
+			admin_interface.current_menu = "delete_log"
+		if("page_deleted_1to1")
+			admin_interface.last_menu = admin_interface.current_menu
+			admin_interface.current_menu = "deleted_talks"
+		if("page_access_management")
+			admin_interface.last_menu = admin_interface.current_menu
+			admin_interface.current_menu = "access_management"
+		if("page_maint_management")
+			admin_interface.last_menu = admin_interface.current_menu
+			admin_interface.current_menu = "maintenance_management"
+
+		// -- 1:1 Conversation -- //
+		if("new_conversation")
+			var/datum/ares_record/talk_log/convo = new(interface.last_login)
+			convo.conversation += "[MAIN_AI_SYSTEM] at [worldtime2text()], 'New 1:1 link initiated. Greetings, [interface.last_login].'"
+			datacore.records_talking += convo
+
+		if("clear_conversation")
+			var/datum/ares_record/talk_log/conversation = locate(params["active_convo"])
+			if(!istype(conversation))
+				return FALSE
+			var/datum/ares_record/deleted_talk/deleted = new
+			deleted.title = conversation.title
+			deleted.conversation = conversation.conversation
+			deleted.user = MAIN_AI_SYSTEM
+			datacore.records_deletion += deleted
+			datacore.records_talking -= conversation
+
+		if("fake_message_ares")
+			var/message = tgui_input_text(user, "What do you wish to say to ARES?", "ARES Message", encode = FALSE)
+			if(message)
+				interface.message_ares(message, user, params["active_convo"], TRUE)
+		if("ares_reply")
+			var/message = tgui_input_text(user, "What do you wish to reply with?", "ARES Response", encode = FALSE)
+			if(message)
+				interface.response_from_ares(message, params["active_convo"])
+				var/datum/ares_record/talk_log/conversation = locate(params["active_convo"])
+				var/admin_log = SPAN_STAFF_IC("<b>ADMINS/MODS: [SPAN_RED("[key_name(user)] replied to [conversation.user]'s ARES message")] [SPAN_GREEN("via Remote Interface")] with: [SPAN_BLUE(message)] </b>")
+				for(var/client/admin in GLOB.admins)
+					if((R_ADMIN|R_MOD) & admin.admin_holder.rights)
+						to_chat(admin, admin_log)
+
+		if("read_record")
+			var/datum/ares_record/deleted_talk/conversation = locate(params["record"])
```

Comment:
This needs an istype check to validate user input actually located the datum type you wanted.